ebd-all-minilm/main.py

```diff
@@ -1,5 +1,6 @@
 import logging
 
+import tqdm  # noqa
```

Comment:
we can remove this since we already doing `--copy-metadata` for this

Reply:
Removed :)